app/src/androidTest/java/com/omen273/crossLingo/CheckStarCountAfterRemovingInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.omen273.crossLingo
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import org.junit.Test
+
+class CheckStarCountAfterRemovingInstrumentedTest: SolveCrossword() {
+
+    @Test
+    fun solveCrosswordInstrumentedTest() {
+        val START_STAR_NAMBER = 10
+        waitForCondition("Stars number checking", {START_STAR_NAMBER  == readConfig()})
+        crossword = generateCrossword()
+        loadFirstCrossword()
+        solve()
+        Espresso.onView(ViewMatchers.withText(R.string.remove))
```

Comment:
what about reset?

Reply:
It had worked before. I'd prefer to add more instrumented tests in other tickets